controllers/automation/policyautomation_controller.go

```diff
@@ -199,6 +199,105 @@ func (r *PolicyAutomationReconciler) Reconcile(
 			} else {
 				log.Info("All clusters are compliant. Doing nothing.")
 			}
+		} else if policyAutomation.Spec.Mode == "everyEvent" {
```

Comment:
If there is `DelayAfterRunSeconds` set and the cluster went from non-compliant -> compliant -> noncompliant, then you'll need to requeue the request so that the `Reconcile` method is called after the `DelayAfterRunSeconds` has elapsed so that the automation is triggered again.